llvm/lib/ObjCopy/MachO/MachOObject.cpp

```diff
@@ -33,6 +33,19 @@ SymbolEntry *SymbolTable::getSymbolByIndex(uint32_t Index) {
       static_cast<const SymbolTable *>(this)->getSymbolByIndex(Index));
 }
 
+void SymbolTable::updateSymbols(function_ref<void(SymbolEntry &)> Callable) {
+  for (auto &Sym : Symbols)
+    Callable(*Sym);
+
+  // Partition symbols: local < defined external < undefined external.
+  auto it_ext = std::stable_partition(
```

Comment:
nit: https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly 
```it_ext -> ExternalSymbolsBegin```